superset-frontend/src/components/ReportModal/index.tsx

```diff
@@ -90,17 +68,16 @@ interface ChartObject {
   triggerQuery: boolean;
   lastRendered: number;
 }
-
 interface ReportProps {
   onHide: () => {};
-  onReportAdd: (report?: ReportObject) => {};
   addDangerToast: (msg: string) => void;
   show: boolean;
   userId: number;
   userEmail: string;
   dashboardId?: number;
   chart?: ChartObject;
   creationMethod: string;
+  props?: any;
```

Comment:
nit, but I don't see this being used anywhere.

Reply:
yes, I will remove it, its a leftover of the rebase.